arch/arm/boot/dts/bcm2711-rpi.dtsi

```diff
@@ -185,17 +185,15 @@
 };
 
 &scb {
-	ranges = <0x0 0x7c000000  0x0 0xfc000000  0x03800000>,
-		 <0x0 0x40000000  0x0 0xff800000  0x00800000>,
-		 <0x6 0x00000000  0x6 0x00000000  0x40000000>,
-		 <0x0 0x00000000  0x0 0x00000000  0xfc000000>;
-	dma-ranges = <0x0 0x00000000  0x0 0x00000000  0xfc000000>,
-		     <0x1 0x00000000  0x1 0x00000000  0x80000000>,
-		     <0x1 0x80000000  0x1 0x80000000  0x80000000>;
+	ranges = <0x0 0x7c000000  0x4 0x7c000000  0x0 0x04000000>,
+		 <0x0 0x40000000  0x4 0xc0000000  0x0 0x00800000>,
+		 <0x6 0x00000000  0x6 0x00000000  0x0 0x40000000>,
+		 <0x0 0x00000000  0x0 0x00000000  0x0 0x80000000>;
```

Comment:
Oops. I'll fix the size here.

Reply:
Actually, what is this supposed to mean? There seems to be overlap in the bus addresses.

Reply:
Which bit are you questioning? I can't see a problem.

Reply:
The one that maps 0x0 -> 0x0. I accidentally changed the size from 0xfc000000 to 0x80000000.

It looks like that range overlaps with the first two. Should it even be there? None of the nodes inside mention them, and none of the other nodes (except memory) define those ranges.

Reply:
It's probably unnecessary - delete it.